dev/docker-compose.yml

```diff
@@ -1,35 +1,98 @@
 services:
   zimfarm-db:
-    image: postgres:15.2-bullseye
+    image: postgres:17.3-bookworm
+    container_name: zimarm-postgresdb
     ports:
-      - 127.0.0.1:5432:5432
+      - 127.0.0.1:2345:5432
```

Comment:
We should keep 5432 for the time being. Until https://github.com/openzim/overview/issues/24 is addressed.

Reply:
I don't get what you mean by get rid of the compose file here and "keeping things simpler"? How can we help developers and maintainers quickly setup the whole stack if we do not provide a docker-compose file anymore?

Reply:
If we add a link that points to the compose file at zimfarm, then, they can start their service alongside the zimfarm own. Assuming the ports are going to be standardized, it would be a simple case of adding `zimfarm_api_url` to `http:://localhost:<port>`. So, all the services can run independently.